src/Transaction.java

```diff
@@ -27,7 +39,21 @@ public String getDescription() {
         return description;
     }
 
+    public Date getDate() {
+        return date;
+    }
+
     public long getMoney() {
         return money;
     }
+
+    public String getSummary() {
+        if (money == 0) {
+            return "No data.";
+        }
+        if (money <0) {
+            return "You have spent £" + (money * -1);
+        }
+        return "You have £" + money;
```

Comment:
GREEN: If you've got 2 transactions of £10 each, then the last transaction will say `"You have £10"` even though you've got $£20.